scripts/manifests/firefox.json

```diff
@@ -53,7 +53,8 @@
 
   "permissions": [
     "storage",
-    "activeTab"
+    "activeTab",
+    "contextMenus"
```

Comment:
New permission(s), you'd need to adjust the `permissions.md` fileand write a sentence of why it is required/what it's used for.

Reply:
OK, will do...